### ISSUE TITLE
Missing words in Onboarding Copy

### ISSUE THREAD

**Issue by golmek** [golemfactory/onboarding#118]
There are strange typos in Onboarding Portal - missing words or misspellings.

**There is:**

![Image](https://github.com/golemfactory/onboarding_staging/assets/126684244/8b7bddaf-1749-4651-a430-0c25650af91a)

**Should be (from Figma Copy of Final designs - proofreading ):**

"GLM is a utility token available on both Polygon and Ethereum blockchains. GLM is used as a currency in peer-to-peer transactions for renting computing power on Golem Network.

In other words, You use GLM tokens to pay for computation power on Golem Network."

"MATIC is a native token of Polygon blockchain, which we recommend as a default blockchain to settle Golem payments.

You need MATIC to be able to cover Polygon transaction fees and as a result to be able to send GLM payments."

Other example:

![Image](https://github.com/golemfactory/onboarding_staging/assets/126684244/649891c2-cdfa-40bc-bf76-4e49be3cabfb)

**Comment by golmek** [comment]
DONE
